src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -8,9 +8,9 @@ The [Appwrite CLI](https://github.com/appwrite/sdk-for-cli) is a command-line ap
 
 # Installation {% #Installation %}
 
-The CLI is packaged both as an  [npm module](https://www.npmjs.com/package/appwrite-cli)  as well as a  [standalone binary](https://github.com/appwrite/sdk-for-cli/releases/latest)  for your operating system, making it completely dependency free, platform independent and language agnostic.
+The CLI is packaged both as a [npm module](https://www.npmjs.com/package/appwrite-cli) as well as a [standalone binary](https://github.com/appwrite/sdk-for-cli/releases/latest)  for your operating system, making it completely dependency free, platform independent and language agnostic.
```

Comment:
It should be "an" because NPM makes the "eh" sounds like "end," right?

```suggestion
The CLI is packaged both as an [npm module](https://www.npmjs.com/package/appwrite-cli) as well as a [standalone binary](https://github.com/appwrite/sdk-for-cli/releases/latest) for your operating system, making it completely dependency free, platform independent, and language agnostic.
```